components/multi-choice/src/index.js

```diff
@@ -65,11 +66,11 @@ const StyledFieldset = styled('div')(
  * - https://govuk-elements.herokuapp.com/errors/example-form-validation-single-question-radio
  *
  */
-const MultiChoice = ({ meta, label, children, hint, ...props }) => (
+const MultiChoice = ({ meta, label, children, hint, language, ...props }) => (
   <StyledFieldset error={meta.touched && meta.error} {...props}>
     <LabelText>{label}</LabelText>
     {hint && <HintText>{hint}</HintText>}
-    {meta.touched && meta.error && <ErrorText>{meta.error}</ErrorText>}
+    {meta.touched && meta.error && <ErrorText><VisuallyHidden> {language === 'cy'? 'Gwall: ' : 'Error: '}</VisuallyHidden>{meta.error}</ErrorText>}
```

Comment:
Instead of passing language, I might have had the parameter called 'errorPrefix' and allowed anything to be passed in.  The advantages of this would be that it's not limited to just 'Gwall' or 'Error', and it would default to nothing (the current behaviour) if the parameter is not passed in making it less of a breaking change.
The way you have it here will be fine for now though if we don't have time to update it.